src/database/testcases.tsx

```diff
@@ -0,0 +1,196 @@
+import { deleteFacility, saveFacility } from "./index";
+
+// 1. Full Form Submission - New Facility
+//
+// Note: some fileds get submitted along with the form submission
+// that aren't actually valid model inputs (i.e. localeDataSource)
+saveFacility({
+  name: "Ascension Correctional Facility",
+  modelInputs: {
+    stateCode: "Louisiana",
+    countyName: "Ascension",
+    rateOfSpreadFactor: "moderate",
+    usePopulationSubsets: true,
+    facilityOccupancyPct: 0.85,
+    facilityDormitoryPct: 0.25,
+    hospitalBeds: 24,
+    localeDataSource: {},
+    totalIncarcerated: 480,
+    ageUnknownPopulation: 305,
+    ageUnknownCases: 3,
+    confirmedCases: 451,
+    staffCases: 3,
+    staffPopulation: 37,
+    plannedReleases: [
+      {
+        date: "2020-04-24T04:00:00.000Z",
```

Comment:
This is how these `date` values come in from the form.  They are stored as a `string` in Firestore which I'm not sure I'm crazy about, but as you can see from the screenshot attached with this PR's opening comment, the "Planned Releases" fields are updated properly.